src/test/osd/osd-scrub-repair.sh

```diff
@@ -17,6 +17,12 @@
 source $(dirname $0)/../detect-build-env-vars.sh
 source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
 
+if [ `uname` = FreeBSD ]; then
+    use_bluestore=false
```

Comment:
@tchaikov 
I do not know enough about erasure and/or the tests, but the overwrite tests actually use bluestore OSDs.  So there is a relation between bluestore and overwrite.
But the problem is bluestore, so I'd still like to call the setting use_bluestore.
However I'll augment the commit and PR text.